configs/_base_/models/mask_rcnn_r50_fpn.py

```diff
@@ -117,4 +117,5 @@
             score_thr=0.05,
             nms=dict(type='nms', iou_threshold=0.5),
             max_per_img=100,
-            mask_thr_binary=0.5)))
+            mask_thr_binary=0.5,
+            rescale_mask_to_input_shape=True)))
```

Comment:
Seems we can add a comment here to indicate this argument only works when exporting to ONNX or support it in normal forward to be consistent.

Reply:
Changed the parameter name from `rescale_mask_to_input_shape` to `rescale_mask_to_input_shape_in_onnx`. I think this should improve readability and should not be misleading when using normal forward.

Reply:
Kindly ping @jshilong